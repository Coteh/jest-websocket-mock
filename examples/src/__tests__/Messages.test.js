import React from "react";
import renderer from "react-test-renderer";
import makeStore from "../store";
import { actions } from "../store/reducer";
import Messages from "../Messages";

describe("The Messages component", () => {
  it("renders the list of sent and received messages", () => {
    const store = makeStore();
    store.dispatch(actions.storeReceivedMessage("ping"));
    store.dispatch(actions.storeSentMessage("pong "));
    store.dispatch(actions.storeReceivedMessage("Oh hi Mark"));
    const component = renderer.create(<Messages store={store} />);
    expect(component).toMatchSnapshot();
  });
});
