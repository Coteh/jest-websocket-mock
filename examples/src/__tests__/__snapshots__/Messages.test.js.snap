// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Messages component renders the list of sent and received messages 1`] = `
<div
  className="Messages"
>
  <div>
    (
    received
    ) 
    ping
  </div>
  <div>
    (
    sent
    ) 
    pong 
  </div>
  <div>
    (
    received
    ) 
    Oh hi Mark
  </div>
</div>
`;
